front-end/src/app/reports/transactions/transaction-list/transaction-list-table-base.component.ts

```diff
@@ -335,6 +336,17 @@ export abstract class TransactionListTableBaseComponent extends TableListBaseCom
     return '';
   }
 
+  private canDelete(transaction: Transaction): boolean {
+    if (
+      transaction.transaction_type_identifier !== 'LOAN_REPAYMENT_MADE' &&
```

Comment:
since loan and debt repayments can take the form of any receipt or disbursement, we might want an allowlist instead:
```
if (
!(transaction.transaction_type_identifier in ["LOAN_RECEIVED_FROM_INDIVIDUAL", "LOAN_RECEIVED_FROM_BANK", "LOAN_BY_COMMITTEE", "DEBT_OWED_BY_COMMITTEE", "DEBT_OWED_TO_COMMITTEE"]) &&
(transaction.loan_id || transaction.debt_id)
)
```